user/contactmethod/contactmethod_test.go

```diff
@@ -27,10 +27,35 @@ func TestContactMethod_Normalize(t *testing.T) {
 	invalid := []ContactMethod{
 		{Name: "abcd", Type: TypeSMS, Value: "+15555555555"},
 	}
+	validIndia := []ContactMethod{
```

Comment:
Add all these to the `valid` slice instead of separate ones. You can add an in-line comment for a hint. Same for `invalid`

```go
valid := []ContactMethod{
  { ... } // US
  { ... } // India
```